### PR TITLE
Views

### PR DESCRIPTION
**Pull request by TEMESGEN-G25**
- Implement the design from the sneak peek wireframes.
- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,10 @@
...
+<h1>Users</h1>
+
+<div id="users">
+<% @users.each do |user| %>
+    <%= render user %>
+    <p>
+      <%= link_to "Show this user", user %>
+    </p>
+  <% end %>
+</div>
```
Comment (CollinsTatang) [inline app/views/users/index.html.erb]: - [ ] Please your user index file should match the given `http://127.0.0.1:3000/` [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES)
**User Wireframe**
user_index
**Your Styling**
youUserDesign

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,15 @@
...
+    <p>
+    <strong>Title:</strong>
+    <%= post.title %>
+  </p>
+
+  <p>
+    <strong>Text:</strong>
+    <%= post.text %>
+  </p>
+<% end %>
```
Comment (CollinsTatang) [inline app/views/users/show.html.erb]: - [ ] Please your user show file should match the given routes `http://127.0.0.1:3000/users/1` [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES)

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,7 @@
-<h1>This is a list of all the posts in the app.</h1>
+<h1>Posts</h1>
+
+<div id="posts">
+  <% @posts.each do |post| %>
+    <%= render post, comments: post.comments %>
+  <% end %>
+</div>
```
Comment (CollinsTatang) [inline app/views/posts/index.html.erb]: - [ ] Please your user index file should match the given routes `http://127.0.0.1:3000/users/1/posts` [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES)

File: app/views/posts/show.html.erb [hunk 4]
```diff
@@ -1 +1,4 @@
-<h1>This is a list of all the posts for a specific user.</h1>
+<h1>Post details</h1>
+
+<%= render @post, comments: @comments %>
+
```
Comment (CollinsTatang) [inline app/views/posts/show.html.erb]: - [ ] Please your user index file should match the given routes `http://127.0.0.1:3000/users/1/posts/5` [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES)

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @NatiG25, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Nati, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by TEMESGEN-G25** [comment]
@SajjadAhmad14 @CollinsTatang Thank you for all your reviews.



**Note**: All optional & required changes have been implemented.

**Pull request merged**